server/src/main/java/org/elasticsearch/index/query/SpanOrQueryBuilder.java

```diff
@@ -49,7 +50,7 @@ public SpanOrQueryBuilder(SpanQueryBuilder initialClause) {
         if (initialClause == null) {
             throw new IllegalArgumentException("[" + NAME + "] must include at least one clause");
         }
-        clauses.add(initialClause);
+        addClause(initialClause);
```

Comment:
I think this is not necessary. The only place where we should do the check is inside `fromXContent` method for all span query builders : `SpanOrQueryBuilder`, `SpanTermQueryBuilder` etc.